Python/peephole.c

```diff
@@ -417,7 +419,7 @@ PyCode_Optimize(PyObject *code, PyObject* consts, PyObject *names,
                 }
                 if (h > i + 1) {
                     fill_nops(codestr, i + 1, h);
-                    nexti = find_op(codestr, h);
```

Comment:
Wouldn't be simpler to just add a condition in this line?
```c
nexti = (h < codelen) ? find_op(codestr, h) : codelen;
```

Reply:
That would leave room for the same bug to be introduced in other calls to find_op.

Reply:
I other calls `find_op()` can't be called with `codelen`.

Reply:
Yes. That's why I said "introduced". this prevents the bug from reappearing when the rest of the code changes.